docs/dev_setup.rst

```diff
@@ -84,6 +89,36 @@ Optional Python bindings are provided using `SWIG <http://www.swig.org/>`_.
 To compile the bindings, a `SWIG <http://www.swig.org/>`_ installation is required, as well as a Python 2 or Python 3 installation (including the development packages on `Linux <https://en.wikipedia.org/wiki/Linux>`_ systems, e.g. ``python-dev``).
 Creation of Python bindings can be enabled/disabled at configuration time.
 
+Optional tools
+==============
+
+Ninja
+-----
+
+`Ninja <https://ninja-build.org/>`_ is a replacement for `make`.
+It can be downloaded from `here <https://github.com/ninja-build/ninja/releases>`_.
+Alternatively, on `Ubuntu <https://en.wikipedia.org/wiki/Ubuntu>`_ (and other `Linux <https://en.wikipedia.org/wiki/Linux>`_ distributions), it can be installed using the package manager with the command ``sudo apt install ninja-build``.
+On macOS, it can be installed using `Homebrew <https://brew.sh/>`_ with the command ``brew install ninja``.
+
+clcache / ccache
+----------------
+
+`clcache <https://github.com/frerich/clcache>`_ (on `Windows <https://en.wikipedia.org/wiki/Microsoft_Windows>`_) and `ccache <https://ccache.dev/>`_ (on `Linux <https://en.wikipedia.org/wiki/Linux>`_ and `macOS <https://en.wikipedia.org/wiki/MacOS>`_) are compiler caches.
+They cache compilations, which means that the first time they are used, compilation will be slower than normal.
+However, subsequent compilations will be significantly faster.
+
+`clcache <https://github.com/frerich/clcache>`_ can be downloaded and installed from `here <https://github.com/frerich/clcache/releases/>`_.
+Note that it will only work with paths that do *not* contain spaces.
+So, if you installed the recommended implementation of `LibXml2 <http://xmlsoft.org/>`_, you will need to move it to a location that does not contain spaces.
```

Comment:
The installer will set registry flags that mean it is found by default. So I think this should either be a recommendation to uninstall and re-install to a path without spaces or ensure that the user manually specifies the needed libXML2 paths.

Reply:
Oops, that's obviously what I did (i.e. manually update my `PATH`), but completely forgot to mention it here. Done in commit 3e71ddcb.